dll/win32/msafd/misc/sndrcv.c

```diff
@@ -291,6 +291,14 @@ WSPRecv(SOCKET Handle,
                                    NULL,
                                    0);
 
+    /* Non-blocking sockets must wait until data is available*/
+    if (Status == STATUS_PENDING && Socket->SharedData->NonBlocking)
+    {
+        if (lpErrno)
+            *lpErrno = WSAEWOULDBLOCK;
```

Comment:
Keep original formatting:
```suggestion
    /* Non-blocking sockets must wait until data is available */
    if (Status == STATUS_PENDING && Socket->SharedData->NonBlocking)
    {
        if (lpErrno) *lpErrno = WSAEWOULDBLOCK;
```

Reply:
(If this is new code, it may be better there to use "new" formatting...)